object-manipulation-1/src/set-value.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable no-unused-vars */
+
+function setValue(object, key, value) {
+  for (var property in object) {
+    object[key] = value;
+  }
```

Comment:
You can do this without a loop. Just assign the `value` to the `object` at the property `key`.

Reply:
OK! I just revised both files and pushed them.